s3transfer/__init__.py

```diff
@@ -380,6 +380,12 @@ class MultipartUploader:
         'RequestPayer',
     ]
 
+    COMPLETE_MULTIPART_ARGS = [
```

Comment:
Oh we do not need to update this module. This is module is no longer used in Boto3 and CLI. It was the old s3 logic that boto3 used before migrating over to the transfer manager interface. I think we should just make sure the transfer manager code path is updated to support this and not touch this module or it's tests.

Reply:
Hmm, all of our upload tests that check this are currently written against this implementation 😬

Reply:
I wrote a new test using the right infrastructure to make sure we've got this case covered.